### DISCUSSION TITLE
any support for GPUs in azure or Runpod?

### DISCUSSION THREAD

**Discussion by findalexli** [skypilot-org/skypilot#2167] [General]
I saw that skypilot only supports AWS and GCP at the moment. Any plans currently on the roadmap for azure, Runpod (which has one of the lowest a100 pricing)?

**Comment by Michaelvll** [comment]
Hey @findalexli, we support azure as well. As shown in the Readme, we currently support: AWS, GCP, Azure, IBM cloud, Oracle cloud, Samsung cloud, Lambda cloud for the VMs and AWS, GCP, Cloudflare for the object store.

We are still working on the Runpod support, but if you would like to try some cloud with similar price, Lambda Cloud might be a good choice as well.

**Reply by TomExMachina** [reply]
Thanks for all you do support but Lambda Cloud never has any hardware availability. Upon just checking, all I see available is A10. Sometimes nothing. Runpod always has a good selection, and I think third-parties can rent out their hardware there.
